### PR TITLE
Update bridge refactor

### PR DESCRIPTION
**Pull request by marcos-iov**
Fix so that if a transaction fails to be registered in the Bridge the PowPeg will continue trying to register the remaining transactions.

### CODE REVIEW

File: src/main/java/co/rsk/federate/BtcToRskClient.java [hunk 1]
```diff
@@ -426,9 +467,11 @@ private StoredBlock findBridgeBtcBlockchainMatchingAncestorUsingBlockLocator() t
         return matchedBlock;
     }
 
-    public void updateBridgeBtcTransactions() throws BlockStoreException {
+    protected void updateBridgeBtcTransactions() throws BlockStoreException {
```
Comment (nathanieliov) [inline src/main/java/co/rsk/federate/BtcToRskClient.java:470]:
```suggestion
    protected void updateBridgeBtcTransactions() {
```

File: src/main/java/co/rsk/federate/BtcToRskClient.java [hunk 2]
```diff
@@ -442,143 +485,184 @@ public void updateBridgeBtcTransactions() throws BlockStoreException {
...
+                            bestChainHeight
+                        );
+                    }
                 }
+            } catch (Exception e) {
+                logger.error(
+                    "[updateBridgeBtcTransactions] An error occurred while informing transaction {} to the Bridge. {}",
+                    txHash,
+                    e.getMessage()
+                );
```
Comment (nathanieliov) [inline src/main/java/co/rsk/federate/BtcToRskClient.java:624]: What do you think about moving this to a separate method and making it there easier to follow? For me, it's hard to follow those big block of code,I mean the `try` block.

Comment (kelvinator07) [reply src/main/java/co/rsk/federate/BtcToRskClient.java:624]: I Agree With Nath, We Could Move Some Of This Logic To Separate Methods, It Can Be Handled Now Or We Create Tasks To Handle It Later

**Comment by kelvinator07** [comment]
I'd Also Suggest We Handle The Multiple Try's And Catches In `updateBridge` In The Callees Methods Instead Of Inside The `updateBridge`

File: src/main/java/co/rsk/federate/BtcToRskClient.java (hunk 2 above)
Comment (marcos-iov) [reply src/main/java/co/rsk/federate/BtcToRskClient.java:624]: Thanks for the comments! I'll create a new story to handle this later

**Review by nathanieliov — APPROVED** [review]
